tests/trx_generator/http_client_async.hpp

```diff
@@ -73,7 +73,9 @@ class session : public std::enable_shared_from_this<session> {
       req_.set(http::field::host, host);
       req_.set(http::field::user_agent, BOOST_BEAST_VERSION_STRING);
       req_.set(http::field::content_type, content_type);
+      req_.set(http::field::connection, "close");
       req_.body() = std::move(request_body);
+      req_.keep_alive(false);
```

Comment:
Why these changes?

Reply:
It would if the trx_generator reused a socket. It creates a new connection for each request. We really need to fix that because on my machine I run out of ports.

Reply:
Thanks. For posterity, maybe add a comment that we disable `keep_alive` because the `trx_generator ` doesn't reuse sockets, but this needs to be changed, and then `keep_alive` should be re-enabled.